packages/hoppscotch-cli/src/utils/collections.ts

```diff
@@ -216,7 +215,7 @@ export const collectionsRunnerResult = (
 export const collectionsRunnerExit = (result: boolean) => {
```

Comment:
One interesting thing with this function is that this will never return (as the program will exit when this is done).
TypeScript has a type for this called `never`. You can just mark this function with that so TypeScript can give you better unreachable code analysis.

Ref: https://www.typescriptlang.org/docs/handbook/2/functions.html#never

```suggestion
export const collectionsRunnerExit = (result: boolean): never => {
```